docs/examples/alibi-detect/README.md

```diff
@@ -1,12 +1,14 @@
 # Serving Alibi-Detect models
 
-Out of the box, `mlserver` supports the deployment and serving of `alibi-detect` models. In this example, we will cover how we can create a detector configuration to then serve it using `mlserver`.
+Out of the box, `mlserver` supports the deployment and serving of [alibi-detect](https://docs.seldon.io/projects/alibi-detect/en/latest/index.html) models. Alibi Detect is an open source Python library focused on outlier, adversarial and drift detection. In this example, we will cover how we can create a detector configuration to then serve it using `mlserver`.
 
-## Reference Data and Configuration
+## Fetch reference data
 
-The first step will be to fetch a reference data for an `alibi-detect` model. For that, we will use the [income Classifier example from the `alibi-detect` documentation](https://docs.seldon.io/projects/alibi-detect/en/latest/examples/cd_chi2ks_adult.html)
+The first step will be to fetch a reference data and other relevant metadata for an `alibi-detect` model. 
 
-Install Alibi dependencies for dataset and detector creation
+For that, we will use the [alibi](https://github.com/SeldonIO/alibi) library to get the adult dataset with [demographic features from a 1996 US census](https://archive.ics.uci.edu/ml/datasets/census+income).
+
+**Note**: Install `alibi` dataset dependencies and `alibi-detect` for detector configuration
```

Comment:
Not a big deal, but we could use Myst annotations on this one, e.g.
````
```{note}
Install `alibi` dataset dependencies and `alibi-detect` for detector configuration
```
````

, which would render a block like the one [here](https://mlserver.readthedocs.io/en/latest/examples/custom/README.html#deploying-our-custom-image)

Reply:
Fixed now in 8cdec90